README.md

```diff
@@ -1 +1,109 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+JSON is a widely use data format. When it comes to Node applications, developers like to use JSON file to store various data such as:
+
+* Configurations
+* Seed data (initial data to populate the development or test database)
+* Settings
+* Plugins for frameworks
+
+Let's implement code which reads a JSON file and parses the file data into JavaScript object (`read.js`). The `customers.json` has information about customers including their emails, balance, phone numbers and addresses. Please don't share this information with anyone.
+
+```js
+var fs = require('fs')
+fs.readFile('./customers.json', {encoding: 'utf8'}, function(error, jsonString){
+  try {
+    var data = JSON.parse(jsonString)
+  } catch(e) {
+    console.log(e)
+  }
+  console.log(data.length, data[2].email)
+})
+```
+
+The fact that we can access property `data[2].email` tells us that `data` is a JavaScript/Node object and not a JSON string. The file has an array that's why we can get its length and access the third item. The items have property `email` among others.
```

Comment:
This is really easy to miss (hence the two comments about it below) -- I would move this down to the section where students are actually interacting with the data.